### PR TITLE
Feat/arbitrum

### PR DESCRIPTION
**Pull request by aminlatifi**
- https://github.com/Giveth/Roadmap/issues/100
- https://github.com/Giveth/impact-graph/issues/1284
- https://github.com/Giveth/impact-graph/issues/1285
- https://github.com/Giveth/impact-graph/issues/1288

### CODE REVIEW

**Review by CarlosQ96 — APPROVED** [review]
LGTM Amin thanks!

File: src/services/chains/index.test.ts [hunk 1]
```diff
@@ -610,6 +610,44 @@ function getTransactionDetailTestCases() {
...
+      toAddress: '0x5c66fef6ea22f37e7c1f7eee49e4e116d3fbfc68',
+      amount,
+      timestamp: 1708342629,
+    });
+    assert.isOk(transactionInfo);
+    assert.equal(transactionInfo.currency, 'ETH');
+    assert.equal(transactionInfo.amount, amount);
+  });
+
+  it('should return transaction detail for normal transfer on CELO Alfajores', async () => {
```
Comment (mohammadranjbarz) [inline src/services/chains/index.test.ts]: @aminlatifi  Please change the description from Celo to Arbitrum

**Review by mohammadranjbarz — CHANGES_REQUESTED** [review]
Thanks @aminlatifi 
It seems good to me, but there are some things that should add Arbitrum networkId there:
1. `qfRoundTab.ts`
2. `tokenTab.ts`

**Comment by aminlatifi** [comment]
> Thanks @aminlatifi It seems good to me, but there are some things that should add Arbitrum networkId there:
> 
> 1. `qfRoundTab.ts`
> 2. `tokenTab.ts`

I had added already at some points, which part do you mean?

**Review by jainkrati — APPROVED** [review]

**Pull request merged**